src/components/common/MemosTable/AddReply/Reply.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+import FormInput from '../../FormInput';
+import './Reply.css';
+
+const initialValues = {
+  content: '',
+};
+
+function ReplyInput(props) {
+  const [formValues, setFormValues] = useState(initialValues);
+
+  const onChange = e => {
+    setFormValues({
+      ...formValues,
+      [e.target.name]: e.target.value,
+    });
+  };
+
+  return props.trigger ? (
+    <div className="popup">
+      <div className="popup-inner">
+        <h1>Add a comment:</h1>
+        <>
+          <label htmlFor={props.labelId}>{props.labelId}</label>
+          <FormInput
+            name="content"
+            type="text"
+            onChange={onChange}
+            placeholder="insert comment here"
+          />
+        </>
+        <br />
+        <button className="submit" onClick={() => console.log('submit')}>
```

Comment:
Check out Ant design buttons for styling of buttons and other pop-ups. It'll help to make the styling more consistent with the rest of the application.